src/controllers/trainee/routes.ts

```diff
@@ -1,13 +1,17 @@
 import { Router } from 'express';
-
 import TraineeController from './Controller';
+import validationHandler from '../../libs/validationHandler';
+import validation from './validation';
+const traineeRouter = Router();
+import  authMiddleWwre from '../../libs/routes/authMiddleware';
+
 
-const TraineeRouter = Router();
+traineeRouter.route('/')
+    .get(authMiddleWwre('getUser', 'read'), validationHandler( validation.get), TraineeController.get)
+    .post(authMiddleWwre('getUser', 'write'), validationHandler( validation.create ), TraineeController.create)
+    .put(authMiddleWwre('getUser', 'update'), validationHandler(validation.update ), TraineeController.update)
+    .delete(authMiddleWwre('getUser', 'delete'), validationHandler( validation.delete ), TraineeController.delete);
+// traineeRouter.route('/:id').delete(validationHandler( validation.delete ), TraineeController.delete);
```

Comment:
This should not be commented as it's a valid route for delete since you need id as part of param. Delete above one that has no :/id params.

Reply:
ok sir